recipe/meta.yaml

```diff
@@ -27,6 +27,10 @@ requirements:
     - {{ compiler('cxx') }}
     - file  # [arm64]
   host:
+    - azure-core-cpp
+    - azure-identity-cpp
```

Comment:
```suggestion
```
TileDB does not currently depend on `azure-identity-cpp`. It will after TileDB-Inc/TileDB#4126 gets merged.

Reply:
Good to know. I knew that azure-identity-cpp was one of the Azure libraries we needed to support TileDB, but I didn't realize that its funtionality hadn't been merged yet. Thanks!